apps/prairielearn/assets/scripts/examTimeLimitCountdown.ts

```diff
@@ -1,15 +1,48 @@
+import { Toast } from 'bootstrap';
+
 import { onDocumentReady, decodeData, parseHTMLElement } from '@prairielearn/browser-utils';
 import { html } from '@prairielearn/html';
 
 import { saveQuestionFormData } from './lib/confirmOnUnload.js';
 import { setupCountdown } from './lib/countdown.js';
 
+function showWarningPopup(message: string) {
```

Comment:
I believe this leaves us with the potential to show multiple warnings at the same time, if e.g. server updates fail multiple times or if we hit the 60 second mark and then the update fails. I guess I'd prefer to mount this HTML once into the DOM and then update the text and/or re-show the toast when something changes. Does that make sense?

Reply:
Addressed